pkg/kudoctl/util/repo/repo.go

```diff
@@ -51,7 +51,8 @@ func NewRepositories() *Repositories {
 }
 
 // GetConfiguration returns a RepoName Config for a name or nil
-func (r Repositories) GetConfiguration(name string) *Configuration {
+func (r *Repositories) GetConfiguration(name string) *Configuration {
+	fmt.Printf("%v\n", r.Repositories)
```

Comment:
a debug remnant?
```suggestion
```

Reply:
yeah....  I separated my prs into 2... it is removed off the 2nd.   missed on this one.  not intended for master.